cmd/kata-pkgsync/config-example.yaml

```diff
@@ -0,0 +1,34 @@
+obsprojects:
```

Comment:
As this is YAML, it would be useful to:

- Add the standard copyright header to keep the CI happy.
- Add a comment at the top of the file explaining that the user will need to change some of the values. For the architectures list, it might be useful to state what format the architecture name is in (`uname -m`?) to make updating it less error prone.

In fact, why not just add a `description:` field to each element like we do in https://github.com/kata-containers/runtime/blob/master/versions.yaml so you can explain each part of the file as the user reads it?